File: Zeze/Raft/RocksRaft/LogMap.cs
using System;
using System.Collections.Generic;
using System.Collections.Immutable;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Zeze.Serialize;

namespace Zeze.Raft.RocksRaft
{
	public abstract class LogMap<K, V> : LogBean
	{
		internal ImmutableDictionary<K, V> Value { get; set; }

		public override void Collect(Changes changes, Bean recent, Log vlog)
		{
			throw new NotImplementedException($"Collect Not Implement.");
		}
	}
}

File: ZezeJava/ZezeJava/src/main/java/Zeze/Raft/RocksRaft/LogMap.java
package Zeze.Raft.RocksRaft;

public abstract class LogMap<K, V> extends LogBean {
	private org.pcollections.PMap<K, V> value;

	public LogMap(int typeId) {
		super(typeId);
	}

	public LogMap(String typeName) {
		super(typeName);
	}

	public final org.pcollections.PMap<K, V> getValue() {
		return value;
	}

	public final void setValue(org.pcollections.PMap<K, V> value) {
		this.value = value;
	}

	@Override
	public void collect(Changes changes, Bean recent, Log vlog) {
		throw new UnsupportedOperationException("Collect Not Implement.");
	}
}
